index.js

```diff
@@ -9,7 +9,7 @@ See the accompanying LICENSE file for terms.
 var isRegExp = require('util').isRegExp;
 
 // Generate an internal UID to make the regexp pattern harder to guess.
-var UID                 = Math.floor(Math.random() * 0x10000000000).toString(16);
+var UID                 = Math.floor(Math.random() * 0x10000000000).toString(36);
```

Comment:
This is the max radix, and the largest random number that produces an 8 char string. I need 8 chars so I can wrap it with `@` special char to end up with a 10 char string which is the max length for `space` param to `JSON.stringify()`.